### DISCUSSION TITLE
DateTime picker

### DISCUSSION THREAD

**Discussion by JGaudray** [TomSchimansky/CustomTkinter#1976] [Ideas]
Hello,
Is there any plan to release a date/time picker ? I was a bit struggling with that and had to find solutions in Tkinter a couple of weeks ago. Thank you :)

**Comment by DanielJOHara** [comment]
Have you tried tkcalendar it has a DateEntry widget. It does not have the customtkinter style but works..

**Comment by Akascape** [comment]
@JGaudray 
For selecting time, you can use Tktimepicker: 
https://github.com/PaulleDemon/tkTimePicker

![NavyBlue](https://github.com/PaulleDemon/tkTimePicker/blob/master/ReadMeImages/NavyBlue.png?raw=True)

For selecting a date, you can use tkcalender or this ctkcalender: https://github.com/ZikPin/CTkDataVisualizingWidgets

![image](https://github.com/ZikPin/Custom-widgets-for-CTk/assets/65452275/b27c41e8-c5bb-4788-92ad-f1e39bc6ab49)

To explore more widgets like this, check this out: https://github.com/Akascape/tkinter-toolkit

**Comment by JGaudray** [comment]
Thanks for the answers! I found also the CTkDatePicker quite interesting:
https://github.com/maxverwiebe/CTkDatePicker

![image](https://github.com/user-attachments/assets/ac24954a-3c16-4c00-bccb-e4182168259b)
